EcatSlaveCommand s1
EcatSlaveCommand s2
EcatSlaveCommand s3
EcatSlaveCommand s4
EcatSlaveCommand s5
EcatSlaveCommand s6
EcatSlaveCommand s7
EcatSlaveCommand s8
EcatSlaveCommand s9
EcatSlaveCommand s10
---
bool success